docs/source/graph-construction/gs-processing/gpartition/sagemaker.rst

```diff
@@ -0,0 +1,133 @@
+===================================
+Running partition jobs on SageMaker
+===================================
+
+Once the :ref:`distributed processing setup<gsprocessing_distributed_setup>` is complete,
+you can use Amazon SageMaker launch scripts to launch distributed processing jobs with AWS resources.
+To demonstrate the usage of GSProcessing on Amazon SageMaker, we will execute the same output in the
+:ref:`GSProcessing Output<gsp-examining-output>` .
+
+Build the docker image for partition jobs on SageMaker
```

Comment:
Same here, move it later until we decide where to put the docker build process for SageMaker.